code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -198,8 +198,15 @@
 	glass_icon_state = "glass_white"
 	glass_name = "glass of milk"
 	glass_desc = "White and nutritious goodness!"
+	self_consuming = TRUE //so that skeletons can process milk without a liver
 
 /datum/reagent/consumable/milk/on_mob_life(mob/living/carbon/M)
+	if(M.dna.species.type == /datum/species/skeleton || M.dna.species.type == /datum/species/plasmaman) //I mostly copied this check from bone hurting juice (and then modified it to check to see if you ARE a skeleton or a plasmaman instead of if you AREN'T a skeleton or a plasmaman)
+		if(volume > 10)
+			M.reagents.remove_reagent(type, volume - 10)
+			to_chat(M, "<span class='warning'>The excess milk is dripping off your bones!</span>")
+		M.heal_bodypart_damage(1,1, 0)
+		. = 1
```

Comment:
Use true false defines

Reply:
I don't quite understand what you're saying here. Are you saying to use ". = TRUE" instead of ". = 1"?